### PR TITLE
Update for FW v5 compatibility

### PR DESCRIPTION
**Pull request by moloko**

### CODE REVIEW

**Review by guywillis — APPROVED** [review]

**Comment by moloko** [comment]
**Note:** please don't merge yet. If 442f687 fixes compatibility with the AAT (see [this forum post](https://community.adaptlearning.org/mod/forum/discuss.php?d=3860#p14101)) I'll need to make the same fix in master then re-release a v2/3/4-compatible version before merging and releasing this

**Pull request merged**
